### PR TITLE
Don't confuse different files with the same title in Google Drive CDN

### PR DESCRIPTION
**Pull request by jikamens**
The Google Drive CDN export code was assuming that files in the same
chunk with the same title were the same file. This was causing all
sorts of problems, such as attempting to delete files multiple times,
attempting to access files that had been deleted, uploading files that
had already been uploaded repeatedly, comparing local files with the
wrong uploaded files to determine whether they needed to be uploaded
again, etc.

I fixed this by adding a new "path" property to uploaded files with
the remote path in it, and using this new property to disambiguate
files with the same remote title.

Note that the first time an export is run after this patch is applied,
all files will be uploaded again, even files that haven't changed,
because they don't have the "path" property yet.

Thank you for proposing a fix!  Please provide a single PR for each bug fix/improvement/new feature. 
Make sure you describe the issue, provide a use case, and elaborate on why your solution is an adequate one.

In Summary:

0. Single PR for a single problem.
1. What is the problem?
2. Where is the cause?
3. How was it solved?

Don't forget to update README.md changleog section and the Wiki Changelog page with a new row.

### CODE REVIEW

**Comment by jikamens** [comment]
I've done some other work on this, so I'm going to close this PR and submit a new one which is cleaned up.
